### DISCUSSION TITLE
why the Variables List has $dark-page, but when use bg-dark-page not work?

### DISCUSSION THREAD

**Discussion by little3201** [quasarframework/quasar#17423] [General - Components / Directives / etc]
why the Variables List has $dark-page, but when use bg-dark-page not work?

**Comment by yusufkandemir** [comment]
That's because `dark-page` is not a brand color:
https://quasar.dev/style/color-palette#brand-colors
It's only for configuring the page background color when the dark mode is active.

**Reply by little3201** [reply]
But in dark mode, the `<q-layout>`, `<q-page>`, background color is not dark, it is transparent. Need to give `<q-page>` or `<q-layout>` background-color like 'bg-dark', or Did I using that error?

note: I set a variable color ($dark-page: '#121212') in quasar.variable.css

**Reply by yusufkandemir** [reply]
It's hard to understand your message. I suggest translating from your native language to English using a tool.

**Reply by little3201** [reply]
Sorry, I didn't notice the issue in the comment (the tag wasn't formatted with the code, it wasn't shown), I've updated the description of the issue

**Reply by yusufkandemir** [reply]
It works fine:
https://stackblitz.com/edit/quasarframework-hnj38d?file=quasar.config.js:L89,src/css/quasar.variables.scss:L20

**Reply by little3201** [reply]
Hmm, you're right, I'm not using it right, set ```
framework: {
dark: 'auto'
}
```, need to use `$q.dark.set(true)`, and directly use `v-model="$q.dark.isActive"` part of the component to take effect`<q-page>`, but it won't set dark background-color.
